### PR TITLE
add link to docs.felt.dev

### PR DESCRIPTION
**Pull request by ryanatkn**

### CODE REVIEW

File: src/routes/DocsLink.svelte [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<a class="panel" href="https://docs.felt.dev/">
+	<div class="text">
+		<div>the</div>
+		<div>docs</div>
+	</div>
+	<div class="heart">🗝</div></a
```
Comment (greatbacon) [inline src/routes/DocsLink.svelte:6]: should we change this "heart" to like, icon or something to mitigate copy-pasta proliferation?

File: src/routes/TimelineLink.svelte [hunk 2]
```diff
@@ -1,4 +1,8 @@
-<a class="panel" href="/timeline">
+<script lang="ts">
+	import {base} from '$app/paths';
+</script>
+
+<a class="panel" href="{base}/timeline">
 	<div class="icon">🗺</div>
```
Comment (greatbacon) [inline src/routes/TimelineLink.svelte:6]:
```suggestion
	<div class="icon">⏳</div>
```
Since we'd been discussing this on Felt

**Review by greatbacon — APPROVED** [review]

File: src/routes/DocsLink.svelte (hunk 1 above)
Comment (ryanatkn) [reply src/routes/DocsLink.svelte:6]: good catch I'll try to make generic everywhere

File: src/routes/+page.svelte [hunk 3]
```diff
@@ -60,4 +75,10 @@
 		max-width: var(--width_md);
 		margin-bottom: var(--spacing_xl5);
 	}
+	.hourglass::before {
+		content: '⏳';
+	}
+	.hourglass:hover::before {
+		content: '⌛';
+	}
```
Comment (ryanatkn) [inline src/routes/+page.svelte:83]: I put this in to see how it feels, I'm alright with removing it

**Pull request merged**